internal/app/types/user.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import "github.com/globalsign/mgo/bson"
+
+type User struct {
+	ID       bson.ObjectId   `json:"_id" bson:"_id,omitempty"`
+	Name     string          `json:"name" bson:"name"`
+	Email    string          `json:"email" bson:"email"`
+	Password string          `json:"password" bson:"password"`
+	Avatars  []string        `json:"avatar" bson:"avatars"` // arr path image
+	Gender   string          `json:"gender" bson:"gender"`
+	Country  string          `json:"country" bson:"country"`
+	Hobby    []Hobby         `json:"hobby" bson:"hobby"`
+	About    string          `json:"about" bson:"about"`
+	Like_id  []bson.ObjectId `json:"like_id" bson:"like_id"`
+	Math     []bson.ObjectId `json:"match" bson:"math"`
```

Comment:
What is this column for?

Reply:
this column for save id users matched. Should change the name to MathID ?.